Solutions/JBoss/Parsers/JBossEvent.txt

```diff
@@ -0,0 +1,16 @@
+// Usage Instructions: 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name as JBossEvent and specifying Legacy Category.
+// This function maps JBoss Enterprise Application Platform events to Azure Sentinel Information Model (ASIM) (https://docs.microsoft.com/en-us/azure/sentinel/normalization).
```

Comment:
Please remove the locale from URL. Please use this url  https://docs.microsoft.com/azure/sentinel/normalization